### DISCUSSION TITLE
Can't upload keystore in "Android code signing"

### DISCUSSION THREAD

**Discussion by Mauge9638** [codemagic-ci-cd/codemagic-docs#2334] [Q&A] [answered]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

1. Select you want to build for Android
2. Try to upload the keystore.jks
![image](https://github.com/codemagic-ci-cd/codemagic-docs/assets/55133846/4fda9bb5-142c-4fbb-8db5-6936e3a8717e)
3. Get this error:
![image](https://github.com/codemagic-ci-cd/codemagic-docs/assets/55133846/1388d5f1-180e-48ef-9eda-269490b8fd1f)

### Expected results

I expect it to get uploaded

### Actual results

But it do not get uploaded, therefore i can't sign my Android App

**Comment by Juliocsreis** [comment]
Same for Google Play Json credentials

**Comment by dtrdic** [comment, marked as answer]
Hello @Mauge9638 @Juliocsreis should be all right now, can you please try again?

**Reply by Mauge9638** [reply]
Yes it works now, thanks! 👍
